config/opensearch.yml.example

```diff
@@ -38,6 +38,10 @@ plugins.security.authcz.admin_dn:
 # BOTH - backend roles are mapped to Security roles mapped directly and via roles_mapping.yml in addition
 plugins.security.roles_mapping_resolution: MAPPING_ONLY
 
+# Specify the default password for the admin user
+# Note: This setting is required for using the default admin user account
+plugins.security.authcz.admin.password:
```

Comment:
I don't think the config file is the right place for a password.  Passwords can be reset; What would happen to this value in the configuration file if that occurred?

Reply:
Yeah no problem.